pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +415,346 @@ def image_threshold(
         # run the algorithm
         _update_alg(alg, progress_bar, 'Performing Image Thresholding')
         return _get_output(alg)
+
+    def fft(self, output_scalars_name=None, progress_bar=False):
+        """Apply a fast Fourier transform (FFT) to the active scalars.
+
+        The input can be real or complex data, but the output is always
+        :attr:`numpy.complex128`. The filter is fastest for images that have
+        the power of two sizes.
```

Comment:
This was definitely correct originally.
```suggestion
        power of two sizes.
```